src/lib/tip-claim-relay.js

```diff
@@ -47,8 +47,7 @@ export default {
 
         if (this.checkAddressMatch(account.publicKey, uniq(addresses))) {
           await this.abortIfZeroClaim(url);
-
-          await axios.post(`${TIP_SERVICE}`, { url, address: account.publicKey });
+          await getters.backendInstance.claimTips({ url, address: account.publicKey });
```

Comment:
You have imported a kind of 'raw' getters, so `getters.backendInstance` is a function, not an object with backend methods.